lib/oedipus_lex.rb

```diff
@@ -556,7 +565,13 @@ def next_token
 
           until ss.eos? or token do
 % if option[:lineno] then
```

Comment:
lineno and column options interfere. I assumed column also assumes lineno, so I added it inside the lineno option